src/views/tutorial-view/utils/get-video-url.ts

```diff
@@ -0,0 +1,16 @@
+import { VideoHostOption } from 'lib/learn-client/types'
+
+const urlBase = {
```

Comment:
This is used in the `serializeContent` function, [copied from Learn](https://github.com/hashicorp/learn/blob/master/lib/get-video-url.ts)